### PR TITLE
[istio] Remove deprecated Istio versions

### PR DESCRIPTION
**Pull request by tarabrind**
## Description

Remove deprecated Istio versions.

## Why do we need it, and what problem does it solve?

No need to support deprecated Istio versions  (1.12 and 1.13).

## Checklist

- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [x] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: istio
type: chore
summary: Remove deprecated Istio versions (1.12 and 1.13).
impact_level: default
```

### CODE REVIEW

File: modules/110-istio/docs/README.md [hunk 1]
```diff
@@ -8,13 +8,10 @@ webIfaces:
...
 |:-------------:|:-----------------------------------------------------------------------------------------------------------------------------:|:------------------------------:|
-|     1.12      |                                 1.19 &#8432;, 1.20 &#8432;, 1.21 &#8432;, 1.22 &#8432;                                        | Deprecated and will be deleted |
-|     1.13      |                                    1.20 &#8432;, 1.21 &#8432;, 1.22 &#8432;, 1.23 &#8432;                                     | Deprecated and will be deleted |
-|     1.16      |                                1.22 &#8432;, 1.23 &#8432;, 1.24 &#8432;, 1.25 &#8432; &#8432;                                 |           Deprecated           |
-|     1.19      |                    1.25 &#8432; &#8432;, 1.26 &#8432; &#8432;, 1.27 &#8432; &#8432;, 1.28 &#8432; &#8432;                     |           Supported            |
+|     1.16      |                                                1.22(!), 1.23(!), 1.24(!), 1.25                                                | Deprecated and will be deleted |
```
Comment (apolovov) [inline modules/110-istio/docs/README.md:11]: @z9r5 The `&#8432;` stuff doesn't work. Please, suggest a nice approach to highlight the versions as non-supported.
image

File: modules/110-istio/docs/README.md [hunk 2]
```diff
@@ -64,6 +61,7 @@ The [AuthorizationPolicy](istio-cr.html#authorizationpolicy) resource is respons
 In other words, if you explicitly deny something, then only this restrictive rule will work. On the other hand, if you explicitly allow something, only explicitly authorized requests would be allowed (however, restrictions will have precedence).
 
 You can use the following arguments for defining authorization rules:
+
```
Comment (apolovov) [inline modules/110-istio/docs/README.md:64]: @z9r5 What do you think about extra newlines?

**Review by apolovov — CHANGES_REQUESTED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

File: modules/110-istio/docs/README.md (hunk 2 above)
Comment (tarabrind) [reply modules/110-istio/docs/README.md:64]:
```shell
root@plodder-master-0:~# kubectl get clusteralerts
NAME               ALERT                                        SEVERITY   AGE     LAST RECEIVED   STATUS
ae9c46e4103fe2a9   D8IstioDeprecatedIstioVersionInstalled       4          3m37s   37s             firing
```

**Comment by tarabrind** [comment]
```shell
root@plodder-master-0:~# kubectl get clusteralerts
NAME               ALERT                                        SEVERITY   AGE     LAST RECEIVED   STATUS
ae9c46e4103fe2a9   D8IstioDeprecatedIstioVersionInstalled       4          3m37s   37s             firing
```

**Review by apolovov — APPROVED** [review]

File: modules/110-istio/docs/README.md (hunk 2 above)
Comment (z9r5) [reply modules/110-istio/docs/README.md:64]: Up to you. It doesn't brake markdown linter rules.

File: modules/110-istio/docs/README.md (hunk 1 above)
Comment (z9r5) [reply modules/110-istio/docs/README.md]: @tarabrind I've pushed the commit. Please check, does it look properly on macOS + Chrome?

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

File: modules/110-istio/docs/README.md (hunk 1 above)
Comment (tarabrind) [reply modules/110-istio/docs/README.md]: Yes, it works correctly now on macOS + Chrome

**Review by nabokihms — APPROVED** [review]

**Pull request merged**